File: scala-gatling/src/gatling/scala/org/openapitools/client/model/InvalidApiKeyError.scala

package org.openapitools.client.model


case class InvalidApiKeyError (
    _result: AnyType,
    _msg: AnyType
)
object InvalidApiKeyError {
    def toStringBody(var_result: Object, var_msg: Object) =
        s"""
        | {
        | "result":$var_result,"msg":$var_msg
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/InvalidApiKeyError.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.AnyType
open OpenAPI.Model.JsonError

module InvalidApiKeyError = 

  //#region InvalidApiKeyError

  [<CLIMutable>]
  type InvalidApiKeyError = {
    [<JsonProperty(PropertyName = "result")>]
    Result : AnyType;
    [<JsonProperty(PropertyName = "msg")>]
    Msg : AnyType;
  }
  
  //#endregion
  